website/screens/components/progress-bar/specs/ProgressBarSpecsPage.tsx

```diff
@@ -0,0 +1,422 @@
+import {
+  DxcTable,
+  DxcStack,
+  DxcList,
+  DxcText,
+} from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import HeadingLink from "../../../common/HeadingLink";
+import specsImage from "./images/progress_specs.png";
+import anatomyImage from "./images/progress_anatomy.png";
+import Figure from "../../../common/Figure";
+import Code from "../../../common/Code";
+import DocFooter from "../../../common/DocFooter";
+
+const ProgressBarSpecsPage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Figure caption="Component design specifications">
+          <Image src={specsImage} alt="Component design specifications" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Anatomy</HeadingLink>
+        <Figure caption="Component anatomy">
```

Comment:
Caption is not used in the original documentation. Remove it and use only `Image`.

Reply:
Done.